wordpress-mobile/WordPress-iOS
# Settings related post

Add the related post settings to site settings.

How to test:

Go to Site->Settings-Related Posts

Change the settings to different values and see if everything works out.
Check in the web to see if the changes are made on the server.
This feature it's only available for sites WP.com or connected with Jetpack.

Needs Review: @jleandroperez
